app/assets/config/manifest.js

```diff
@@ -1,2 +1,2 @@
-//= link_tree ../images
-//= link_directory ../stylesheets .css
+// = link_tree ../images
+// = link_directory ../stylesheets .css
```

Comment:
(These get processed by Sprockets. I'm not sure if the spaces would break that, but I guess us having webpacker means we aren't using Sprockets any more anyway. If that's the case is `manifest.js` still required?)